AnnoyancesFilter/sections/popups.txt

```diff
@@ -119,6 +119,8 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+twitter.com#?##layers > div[class] > div[class] > div[class] > div[class] > div[class] > div[class] > div[class][tabindex][role]:has(> div[class][data-testid] > div[class] > div[class] > div[class] > div[class] > div[class] > div[class] > div[class] > span > span[class]:contains(Log in))
```

Comment:
The first rule does not applied on my end.
What about this?
```
twitter.com#$?#div[data-testid="sheetDialog"]:upward(div[role="group"]) { display: none !important; }
twitter.com#$?#html { overflow: auto !important; padding-right: 0 !important; }
```
If you have account, please test with/without account in new private tabs.